chromium_src/chrome/browser/devtools/url_constants.cc

```diff
@@ -0,0 +1,11 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/devtools/url_constants.h"
+
+const char kRemoteFrontendDomain[] = "devtools.brave.com";
```

Comment:
can we not use the normal static redirect? My concern is that we're changing urls in different places and that makes it harder to find how/where/what we're changing

Reply:
@bridiver - I was trying that first, but it requires a much bigger re-write of the code. 

Currently, the brave-proxy code lives in `brave_proxying_url_loader_factory.cc` which lives in `//brave/browser/net`. To use static redirect we need to invoke `MaybeProxyRequest` from `devtools_instrumentation::WillCreateURLLoaderFactory` which is present in `//content/browser`. 

So for static redirect to work, we'll need to move the proxying code to `//content/` or `//net`.

Reply:
@jumde do you remember if we have an issue for moving proxying code to `//net`? At least `brave_system_request_handler.cc`. I guess this is not the first layering problem with this

Reply:
@iefremov - Filed one: https://github.com/brave/brave-browser/issues/9770 - Thanks for the reminder.